packages/http-server/src/http-server.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/http-server
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {createServer, Server} from 'http';
+import {RestServer, HttpRequestListener} from '@loopback/rest';
+import {AddressInfo} from 'net';
+import * as pEvent from 'p-event';
+
+/**
+ * Object for specifyig the HTTP / HTTPS server options
+ */
+export type HttpOptions = {
+  port: number;
+  host: string | undefined;
+};
+
+/**
+ * HTTP / HTTPS server used by LoopBack's RestServer
+ *
+ * @export
+ * @class HttpServer
+ */
+export class HttpServer {
+  private restServer: RestServer;
+  private httpPort: number;
+  private httpHost: string | undefined;
+  private httpServer: Server;
```

Comment:
Is it necessary to prefix all variables with `http`, considering that they belong to a class that already has `http` in the name?